### ISSUE TITLE
Content migration - add "Deploying to Aerobatic" doc

### ISSUE THREAD

**Issue by amberleyromo** [gatsbyjs/gatsby#8736]
## Issue Context

This issue is part of the content migration project (#8103), which is meant to take existing content and repurpose it so it's easier to find in the docs.

### Motivation

We are doing this because many people have written excellent content that we want to canonize and maintain so more people can find it and benefit from it.

## Steps to resolve this issue

1. Read the Style Guide and follow it in order for your PR to be accepted. Pay special attention to the templates. 
- Style Guide: https://www.gatsbyjs.org/docs/gatsby-style-guide/
- Docs Templates: https://www.gatsbyjs.org/docs/templates/
- Example of a guide to imitate: https://www.gatsbyjs.org/docs/deploying-to-s3-cloudfront

2. Create the "Deploying to Aerobatic" article by creating a file named `deploying-to-aerobatic.md` in [this directory](https://github.com/gatsbyjs/gatsby/tree/master/docs/docs).

3. Copy and paste this section of the deploying doc: https://www.gatsbyjs.org/docs/deploy-gatsby/#aerobatic

4. Make sure you adjust the content to follow the format listed in these resources:
- Docs Templates: https://www.gatsbyjs.org/docs/templates/
- Example of a guide to imitate: https://www.gatsbyjs.org/docs/deploying-to-s3-cloudfront

5. Link the new doc in the docs sidebar file, [doc-links.yml](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/sidebars/doc-links.yaml), under the Deploying section:

```yaml
- title: Deploying to Aerobatic
  link: /docs/deploying-to-aerobatic
```

**Comment by amberleyromo** [comment]
Open PR: https://github.com/gatsbyjs/gatsby/pull/8728

**Comment by drex44** [comment]
hi, can I work on this or the existing PR is good?
I saw there are multiple issues similar to this one and all have pending PRs.

**Comment by amberleyromo** [comment]
Hi @drex44! There indeed is a PR open for this. If there's an open PR for an issue, or someone has commented that they're working on an issue, please check out the other open issues! https://github.com/gatsbyjs/gatsby/issues/8719 

Or, feel free to hop over on the PR and make suggestions to make it better, as well!

**Comment by amberleyromo** [comment]
Closed with #8728
